tea shop/products/products.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="../main.css">
+    <link rel="stylesheet" href="../products.css">
+</head>
+<header>
+    <h2>A quaint little Tea Shop</h2>
+    <nav href="../">Home </nav>
```

Comment:
instead of a nav tag this would probably look something like this... 
`<nav>
<a href="index.html">Home</a>
</nav>`

Reply:
nav is short hand for navigation. 
ok. made all changes! thank you.